embedded-graphics/src/geometry/point.rs

```diff
@@ -666,4 +694,15 @@ mod tests {
 
         assert_eq!(Point::from(&c), Point::new(20, 20));
     }
+
+    #[test]
+    // TODO: Delete this test if component_min/component_max are made public. It will be redundant
+    // with their respective doc comments.
```

Comment:
I've removed all the `TODO` comments. The doc/unit tests are mostly dupes of each other now. Maybe some room for improvement but it can't hurt to double check.